cloud-infrastructure/storage/block-storage/asset/README.md

```diff
@@ -0,0 +1,12 @@
+# Asset
+
```

Comment:
So the idea with the assets are to have a folder per document/file-based asset. 

.. / block-storage / how-to-guide-ABC / files / guide.doc
.. / block-storage / health-check / files / healthcheck.pdf

Each <asset name> folder has a README.md which is templatized as per my Confluence guide page and a LICENSE file. The actual asset binaries are under the files sub-folder.